vosekast_control/Tank.py

```diff
@@ -15,7 +15,6 @@ class Tank():
     FILLED = 1
```

Comment:
That seems inconstist for me. You use strings and numbers for one state. 
I would suggest to use only strings. So `FILLED = 'FILLED'`.

Reply:
ok